Assets/MRTK/Tools/MSBuild/Scripts/UnityProjectInfo.cs

```diff
@@ -27,15 +27,16 @@ public class UnityProjectInfo
         };
 
         /// <summary>
-        /// These package refernces are only for Unity 2019.3+ and shouldn't be included when using older versions
+        /// These package references are only for Unity 2019.3+ and shouldn't be included when using older versions
         /// </summary>
         private static readonly HashSet<string> PackageReferencesUnity2019 = new HashSet<string>()
         {
             "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus",
             "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.Editor",
             "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.Handtracking.Editor",
             "Microsoft.MixedReality.Toolkit.Providers.XRSDK.WindowsMixedReality",
-            "Microsoft.MixedReality.Toolkit.Providers.XRSDK"
+            "Microsoft.MixedReality.Toolkit.Providers.XRSDK",
+            "UnityEngine.SpatialTracking"
```

Comment:
@CDiaz-MS You'll likely need to do something similar for the Leap asmdef update. Probably under the `ExcludedPackageReferences` hashset right above this one, since they won't be specific to Unity 2019.